src/pudl/metadata/resources/ferc1.py

```diff
@@ -822,6 +822,21 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "denorm_plants_utils_ferc1": {
```

Comment:
@zaneselvans should we update how the data dictionaries and documentation is structured to accommodate the new denormalized tables? Maybe have a subheader for normalized and denormalized tables on the data dictionary page?

Reply:
This feels like a larger design question that is maybe out of scope of this PR and has the potential to block converting output tables. I propose we merge this PR as is so folks can start converting output tables. This means our data dictionaries and dataset won't have any categorization for a while. As tables are being converted we can sort out this issue.

Reply:
All `field_namespace` does is indicate what group of overrides should be used when defining field metadata, if the default field information isn't appropriate. It doesn't correlate directly with data source. There's an `eia` field namespace (because we're requiring all field names to mean the same thing in all EIA data), but there's no `eia860` or `eia861` or `eia923` field namespaces.

When would we have an asset that we're shipping as a database table that does not have any metadata associated with it? I don't think we should be adding the raw / interim assets to the data dictionaries. It seems like only the `norm` and `denorm` tables will typically be getting shipped and need to be documented (with the exception of the still unfinished `eia861` tables that are really at the "clean" stage for now, e.g. `clean_sales_eia861` and `clean_advanced_metering_infrastructure_eia861` but `norm_service_territory_eia861` and a couple of others that have actually been finished off. I doubt we'll be able to implement a totally uniform taxonomy of table names but we can try.

I like your filesystem tree! I look forward to getting there!  (Except that `ferc` should probably be `ferc1` -- sadly there's little to no uniformity across the FERC forms so I don't know that we're ever going to have a top-level FERC category)

I agree that we're just gonna have a ton of tables that can be linked to and `CTRL-F` for now. I think it'll be easier to sort them out once we see their natural categories and the weirdos that stand out.

Reply:
Sounds good 👍 I say we figure this out later so we can start converting output tables.

Sorry my comment about assets without metadata was confusing:
> I'm not entirely sure how to implement this right now because not all assets will have metadata.

I imagine whatever categorization we come up with the data dictionaries can be used for the asset group names. Some asset groups have assets that aren't written to the db. For example: `raw_eia860`, `clean_eia923`...